operators/operator_sdk/osdk-pkgman-to-bundle.adoc

```diff
@@ -0,0 +1,14 @@
+[id="osdk-pkgman-to-bundle"]
+= Migrating package manifest projects to bundle format
+include::modules/common-attributes.adoc[]
+:context: osdk-pkgman-to-bundle
+
+toc::[]
+
+If you have an Operator project that was initially created using the package manifest format, which is deprecated, you can use the Operator SDK to migrate the project to the bundle format, which is the preferred packaging format for Operator Lifecycle Manager (OLM) starting in {product-title} 4.6.
```

Comment:
Small consideration to split this into two sentences to break it up some:

> If you have an Operator project that was initially created using the package manifest format, which is deprecated, you can use the Operator SDK to migrate the project to the bundle format. Bundle format is the preferred packaging format for Operator Lifecycle Manager (OLM) starting in {product-title} 4.6.

Reply:
Thank you, I had wrestled with this Frankenstein.